apps/health_alerts/static/js/healthAlertSubscriber.js

```diff
@@ -27,10 +27,6 @@ export default function() {
   function closeBtnClicked() {
     const closeBtn = document.querySelector("#health-alert-subscriber-close-btn-hip");
     closeBtn.addEventListener("click", function () {
-      document.body.classList.remove("modal-is-open-hip");
```

Comment:
Yes it is! I realized that this code was not needed, because when the user hits the close button the page refreshes. The `body` element no longer has `modal-is-open-hip` being applied.

Reply:
🆒